apps/course/src/app/layouts/course-layout/course-layout.component.ts

```diff
@@ -42,6 +44,9 @@ export class CourseLayoutComponent implements OnInit {
     this.courseService.getCurrentCourse().subscribe((course) => {
       if (course) {
         this.isLoading = false;
+      } else {
+        this.isLoading = false;
+        this.platformLocation.back();
```

Comment:
What if the platform / browser history is empty, since I might directly open the course detail route in a new tab? Probably should add alternative route to `/` or similar. 🚀